pkg/kubeapiserver/options/storage_versions.go

```diff
@@ -104,3 +106,14 @@ func (s *StorageSerializationOptions) AddFlags(fs *pflag.FlagSet) {
 		"It defaults to a list of preferred versions of all known groups.")
 
 }
+
+// AllPreferredGroupVersions returns the preferred versions of all registered
+// groups in the form of "group1/version1,group2/version2,..."
+func ToPreferredVersionString(versions []schema.GroupVersion) string {
```

Comment:
is order not important here? or is it an error for versions to contain more than one item in the same group?

Reply:
> is order not important here? or is it an error for versions to contain more than one item in the same group?

Groups only have one version when calling it.